src/components/Sidebar/index.js

```diff
@@ -58,14 +58,33 @@ const Sidebar = () => {
       <div className={isOpen ? sidebar : sidebarClose}>
         <div className={sidebarContentWrapper}>
           <div className={sidebarContent}>
-            <PlaceBox />
-            <ReviewBox />
-            <ReviewInputBox />
+            {places.length ? (
+              places.map(place => <PlaceBox key={place.placeId} />)
+            ) : (
+              <></>
+            )}
```

Comment:
개인적으로 early return을 선호하는데요.

```js
if( !places.length) return null;
```
여기서는 `&&` 을 사용하면 가독성이 더 좋을 것 같습니다. 
```suggestion
         place.length && places.map(place => <PlaceBox key={place.placeId} />)
```